<launch>
  
  <!--Nodes-->  
  <node pkg="gps_odom_tf" type="stanley2.py" name="stanley_control_node" output="screen"  >
    <param name="path" value="$(find gps_odom_tf)/path/highway.txt"/>
    <param name="K" value="0.9"/>
    <param name="L" value="1.04"/>
    <param name="max_steering" value="28.5"/>
    <param name="max_speed" value="7"/>
  </node> 
  
  <node pkg="gps_odom_tf" type="path_viz.py" name="path_visualize_node" output="screen">
    <param name="file_path" value="$(find gps_odom_tf)/path/highway.txt"/>
    <param name="R" value="0.5"/>
  </node>
   
  
</launch>
